doc/how_to_guides/parallel_planning/parallel_planning_tutorial.rst

```diff
@@ -0,0 +1,174 @@
+is called whenever a pipeline produces a solution during the parallel planning process and, if the stopping criterion is met, terminates pipelines that have not found a solution yet.
+
+Example
+-------
+
+The following demo shows how you can configure and use MoveItCpp's parallel planning interface. First,
+run the demo: ::
+
+  ros2 launch moveit2_tutorials parallel_planning_example.launch.py
+
+A complex kitchen scene is loaded and two planning problems are solved. The first one is a small motion of the end effector towards the ground. This problem is likely to be solved by all three
+planners, but with significant differences in the planning time. The second problem is much harder and most likely only the :code:`RRTConnect` planner will succeed. This demo
+suggests that a well-configured parallel planning setup is versatile, and can be used in a broad variety of motion planning problems.
+
+What code is necessary to use parallel planning?
+------------------------------------------------
+
+First, you need to initialize :code:`moveit_cpp` and a planning component that will solve your planning problems. Next, you need to set start state and goal constraints:
+
+.. code-block:: c++
+
+    planning_component_->setGoal(*goal_state);
+    planning_component_->setStartStateToCurrentState();
+
+Additionally, it is necessary to set up the :code:`MultiPipelinePlanRequestParameters`.
+
+.. code-block:: c++
+
+    moveit_cpp::PlanningComponent::MultiPipelinePlanRequestParameters multi_pipeline_plan_request{
+      node_, { "ompl_rrtc", "pilz_lin", "chomp" }
+    };
+
+The constructor of this class will initialize multiple :code:`PlanningRequestParameter` class members based on the config that is provided in the node's
+parameter namespaces :code:`"ompl_rrtc"`, :code:`"pilz_lin"`, and :code:`"chomp"`. To provide these, you can simply extend the :code:`moveit_cpp.yaml` file:
+
+.. code-block:: yaml
+
+    # PlanRequestParameters for the first parallel pipeline
+    ompl_rrtc:
+      plan_request_params:
+        planning_attempts: 1
+        planning_pipeline: ompl
+        planner_id: "RRTConnectkConfigDefault"
+        max_velocity_scaling_factor: 1.0
+        max_acceleration_scaling_factor: 1.0
+        planning_time: 0.5
+
+    # PlanRequestParameters for the second parallel pipeline
+    pilz_lin:
+      plan_request_params:
+        planning_attempts: 1
+        planning_pipeline: pilz_industrial_motion_planner
+        planner_id: "LIN"
+        max_velocity_scaling_factor: 1.0
+        max_acceleration_scaling_factor: 1.0
+        planning_time: 0.8
+
+    # PlanRequestParameters for the third parallel pipeline
+    chomp:
+      plan_request_params:
+        planning_attempts: 1
+        planning_pipeline: chomp
+        max_velocity_scaling_factor: 1.0
+        max_acceleration_scaling_factor: 1.0
+        planning_time: 1.5
```

Comment:
For these comments, maybe add more than just "first, second, and third" and describe them as you did in the fourth one?